asf-site-src/source/documentation/in-progress/precommit/plugins/github.html.md

```diff
@@ -41,8 +41,16 @@ None
 | `--github-api-url=<url>` | REST API URL (for GitHub Enterprise) |
 | `--github-base-url=<url>` | Non-REST API URL (for GitHub Enterprise) |
 | `--github-repo=<repo>` | `username/repository` identifier |
+| `--github-status-use-htmlreport=<bool>` | Use htmlout for Github Status 'Details' link |
 | `--github-token=<token>` | Token used to perform read and write operations |
 
+## HTML Details LInk
+
+By default, if Apache Yetus is under conditions when it would normally write a Github Status (e.g., Jenkins processing
```

Comment:
Can you show an example of the current behavior and (if possible) the htmlreport that will be the new default after this patch? I'm looking at an [HBase PR](https://github.com/apache/hbase/pull/4305) (Yetus running in the context of an Jenkins Multi-branch plugin) and I don't see a "Details" link anywhere in the comment left by the Jenkins build worker. Perhaps my example context doesn't apply to this change?

Reply:
Or perhaps my sample PR doesn't apply at all because HBase isn't yet using the GitHub Checks API...

Reply:
Correct. That PR isn't using the checks API.  You can see in pretty much any Apache Hadoop PR that down at the bottom there is a 'Details' link for 'Apache Yetus'.  That should now point to the HTML report instead of the Jenkins console post this PR.

Reply:
> Apache Yetus 0.12.0

That PR isn't even check API capable...